cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -169,20 +169,12 @@ func (s *Server) GetManager() (utils.AssetManager, error) {
 
 // GetAvailablePackageSummaries returns the available packages based on the request.
 func (s *Server) GetAvailablePackageSummaries(ctx context.Context, request *corev1.GetAvailablePackageSummariesRequest) (*corev1.GetAvailablePackageSummariesResponse, error) {
-	contextMsg := ""
-	if request.Context != nil {
-		contextMsg = fmt.Sprintf("(cluster=[%s], namespace=[%s])", request.Context.Cluster, request.Context.Namespace)
-	}
-
+	contextMsg := fmt.Sprintf("(cluster=[%s], namespace=[%s])", request.GetContext().GetCluster(), request.GetContext().GetNamespace())
```

Comment:
+1. I wondered when I initially saw these why we don't instead use the go fmt built in support for quoting things to do something like:

```suggestion
	contextMsg := fmt.Sprintf("(cluster=%q, namespace=%q)", request.GetContext().GetCluster(), request.GetContext().GetNamespace())
```

But not necessary - just if you agree that'd be nicer (main reason for me is that it's clear that cluster and namespace a string values in the log output, as opposed to being arrays/slices of length 1).